static/js/compose.js

```diff
@@ -538,6 +544,12 @@ exports.transmit_message = function (request, success, error) {
         send_message_ajax(request, success, error);
     }
 };
+function report_content_not_sent(request) {
+    var message_undelivered_id = "" + request.local_id;
+    message_undelivered_id = "div[zid='" + message_undelivered_id.replace( /(:|\.|\[|\]|,|=|@)/g, "\\$1" ) + "']";
```

Comment:
To use the replace function the variable has to be a string, without it the replace function doesnt produce the expected output. For example, it cannot change a float variable(114.14) to the required output so that the jquery doesn't take the period symbol in wrong way.